apps/documentation/src/routes/documentation/tutorial/error-handling.mdx

```diff
@@ -98,5 +98,5 @@ async fn get_all_pokemons(_req: Request, fetch: Client) -> Response {
 }
 ```
 
-If you now try to load a not existing pokemon (`http://localhost:3000/pokemons/tuono-pokemon`) you will
+If you now try to load a not-existing Pokémon (`http://localhost:3000/pokemons/tuono-pokemon`) you will
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Do not mix variants of the same word (‘Pokémon’ and ‘Pokemon’) within a single text. (EN_WORD_COHERENCY)
Suggestions: `Pokemon`
Rule: https://community.languagetool.org/rule/show/EN_WORD_COHERENCY?lang=en-US
Category: MISC
<!-- __reviewdog__:ChA3YzEzY2FhZWVjMDA5MTU5EgxMYW5ndWFnZVRvb2w= -->